src/client.rs

```diff
@@ -167,6 +168,38 @@ impl SafeClient {
         .map(Option::unwrap)
     }
 
+    /// Get information about tokens available on the API
+    #[tracing::instrument(skip(self))]
+    pub async fn tokens(&self) -> ClientResult<TokenInfoResponse> {
+        json_get!(
+            &self.client,
+            TokenInfoRequest::url(self.url()),
+            TokenInfoResponse,
+        )
+        .map(Option::unwrap)
+    }
+
+    /// Get fitered information about tokens available on the API
+    #[tracing::instrument(skip(self, filters))]
+    pub async fn filtered_tokens(
+        &self,
+        filters: impl AsRef<HashMap<&'static str, String>>,
```

Comment:
looking back at this, do you think it'd be good to do something like

`impl IntoIterator<Item = (&'static str, String)>>` for the filters args 🤔 

that makes it easier for users to bring their own query params

Reply:
Yeah I think this is nice for sure!